rs/nns/governance/src/proposals/install_code.rs

```diff
@@ -239,13 +274,28 @@ mod tests {
         assert_eq!(
             InstallCode {
                 canister_id: Some(ROOT_CANISTER_ID.get()),
+                install_mode: Some(CanisterInstallMode::Install as i32),
                 ..valid_install_code.clone()
             }
             .validate(),
             Err(GovernanceError::new_with_message(
                 ErrorType::InvalidProposal,
-                "InstallCode proposal invalid because of InstallCode for root canister is not \
-                 supported yet"
+                "InstallCode proposal invalid because of InstallCode mode Install is not supported for root canister, \
+                consider using HardResetNnsRootToVersion proposal instead"
```

Comment:
I would encourage you to not be so strict on the exact contents of the message. We should avoid rewording causing test failures. Admittedly, in practice, clients sometimes rely on the exact wording, but this kind of condones that.

[Here is an example of an alternative](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/nns/cmc/src/main.rs?L2542-2550).

Reply:
Done.